docs/user-guide/zwe-init-subcommand-overview.md

```diff
@@ -0,0 +1,202 @@
+
+:::info**Required roles:** system programmer
+:::
+
+The command `zwe init vsam` uses the template JCL in `SZWESAMP(ZWECSVSM)` to copy the source template member from `zowe.setup.mvs.hlq.SZWESAMP(ZWECVCSM)` and creates a target JCL member in `zowe.setup.mvs.jcllib(ZWECVSCM)` with values extracted from the `zowe.yaml` file.
+
+For more information about `zwe init vsam`, see [Creating VSAM caching service datasets](./initialize-vsam-dataset).
+
+## Installing Zowe main started tasks (`zwe init stc`)
+
+Execute the subcommand `zwe init stc` to install Zowe main started tasks.
+
+Installation of Zowe main started tasks requires that JCL members for each of Zowe's started tasks be present on the JES proclib concatenation path. 
+
+Once you have completed security configuration, you can install the Zowe main started tasks. 
+
+:::info**Required role:** system programmer
+:::
+
+The JCL members for each of Zowe's started tasks need to be present on the JES proclib concatenation path. The command `zwe init stc` copies these members from the install source location `.SZWESAMP` to the targted PDS specified in the `zowe.setup.dataset.proclib` value `USER.PROCLIB`. The three proclib member names are specified in `zowe.yaml` arguments.  
+
+```
+zowe
+  setup
+    security
+      stcs
+        zowe: ZWESLSTC
+        zis: ZWESISTC
+        aux: ZWESASTC
+```
+
+The `zwe init stc` command uses the `CUST.JCL` LIB data sets as a staging area to contain intermediatory JCL which are transformed version of the originals that are shiped in `.SZWESAMP` with paths, PDS locations, and other runtime data updated.  If you wish to just generate the `CUST.JCLLIB` members without having them copied to  `USER.PROCLIB`, specify `--security-dry-run`.  If the JCL members are already in the target PROCLIB, specify `--allow-overwritten`.   
+
+**Example:**
+
+```
+#>zwe init stc -c ./zowe.yaml
+-------------------------------------------------------------------------------
+>> Install Zowe main started task
+
+Modify ZWESLSTC
+Modify ZWESISTC
+Modify ZWESASTC
+
+Copy IBMUSER.ZWEV2.CUST.JCLLIB(ZWESLSTC) to USER.PROCLIB(ZWESLSTC)
+Copy IBMUSER.ZWEV2.CUST.JCLLIB(ZWESISTC) to USER.PROCLIB(ZWESISTC)
+Copy IBMUSER.ZWEV2.CUST.JCLLIB(ZWESASTC) to USER.PROCLIB(ZWESASTC)
+
+>> Zowe main started tasks are installed successfully.
+#>
+```
+
+## Next steps
+
+After each of the `zwe init` subcommands run successfully, and you completed the security configuration, assigned security permissions to users, and configured the cross memory server (ZIS), you can procede to [starting Zowe](./start-zowe-zos.md).
+
+* For more information about configuring security, see the articles in the section [Configuring security](./configuring-security).
+* For more information about configuring the cross memory server (ZIS), see [Configuring the Zowe cross memory server (ZIS)](./configure-xmem-server.md).
```

Comment:
Major parts of the security and cross memory server configurations is not done by zwe init and so it is to early to proceed to starting Zowe yet, at least swap these lines to first propose to go to security configuration and ZIS config